packages/mobile/src/config/navigation-theme.config.ts

```diff
@@ -0,0 +1,52 @@
+import { ColorPalette } from '@vse-bude/shared';
+import { DefaultTheme, DarkTheme } from '@react-navigation/native';
+
+const COMMON_COLORS = {
+  accent: ColorPalette.YELLOW_100,
+  organ: ColorPalette.GREEN_200,
+  card: ColorPalette.GRAY_100,
+  inputBg: ColorPalette.GRAY_100,
+  dropdownBg: ColorPalette.GRAY_100,
+  circleBg: ColorPalette.GRAY_100,
```

Comment:
We may combine it under some common name.

Reply:
corrected